nl80211/nl80211_defs.go

```diff
@@ -88,8 +88,59 @@ const (
 	CMD_STOP_SCHED_SCAN                 = 0x4c
 	CMD_SCHED_SCAN_RESULTS              = 0x4d
 	CMD_SCHED_SCAN_STOPPED              = 0x4e
-	__CMD_AFTER_LAST                    = 0x4f
-	CMD_MAX                             = 0x4e
+	CMD_SET_REKEY_OFFLOAD               = 0x4f
+	CMD_PMKSA_CANDIDATE                 = 0x50
+	CMD_TDLS_OPER                       = 0x51
+	CMD_TDLS_MGMT                       = 0x52
+	CMD_UNEXPECTED_FRAME                = 0x53
+	CMD_PROBE_CLIENT                    = 0x54
+	CMD_REGISTER_BEACONS                = 0x55
+	CMD_UNEXPECTED_4ADDR_FRAME          = 0x56
+	CMD_SET_NOACK_MAP                   = 0x57
+	CMD_CH_SWITCH_NOTIFY                = 0x58
+	CMD_START_P2P_DEVICE                = 0x59
+	CMD_STOP_P2P_DEVICE                 = 0x5a
+	CMD_CONN_FAILED                     = 0x5b
+	CMD_SET_MCAST_RATE                  = 0x5c
+	CMD_SET_MAC_ACL                     = 0x5d
+	CMD_RADAR_DETECT                    = 0x5e
+	CMD_GET_PROTOCOL_FEATURES           = 0x5f
+	CMD_UPDATE_FT_IES                   = 0x60
+	CMD_FT_EVENT                        = 0x61
+	CMD_CRIT_PROTOCOL_START             = 0x62
+	CMD_CRIT_PROTOCOL_STOP              = 0x63
+	CMD_GET_COALESCE                    = 0x64
+	CMD_SET_COALESCE                    = 0x65
+	CMD_CHANNEL_SWITCH                  = 0x66
+	CMD_VENDOR                          = 0x67
+	CMD_SET_QOS_MAP                     = 0x68
+	CMD_ADD_TX_TS                       = 0x69
+	CMD_DEL_TX_TS                       = 0x6a
+	CMD_GET_MPP                         = 0x6b
+	CMD_JOIN_OCB                        = 0x6c
+	CMD_LEAVE_OCB                       = 0x6d
+	CMD_CH_SWITCH_STARTED_NOTIFY        = 0x6e
+	CMD_TDLS_CHANNEL_SWITCH             = 0x6f
+	CMD_TDLS_CANCEL_CHANNEL_SWITCH      = 0x70
+	CMD_WIPHY_REG_CHANGE                = 0x71
+	CMD_ABORT_SCAN                      = 0x72
+	CMD_START_NAN                       = 0x73
+	CMD_STOP_NAN                        = 0x74
+	CMD_ADD_NAN_FUNCTION                = 0x75
+	CMD_DEL_NAN_FUNCTION                = 0x76
+	CMD_CHANGE_NAN_CONFIG               = 0x77
+	CMD_NAN_MATCH                       = 0x78
+	CMD_SET_MULTICAST_TO_UNICAST        = 0x79
+	CMD_UPDATE_CONNECT_PARAMS           = 0x7a
+	CMD_SET_PMK                         = 0x7b
+	CMD_DEL_PMK                         = 0x7c
+	CMD_PORT_AUTHORIZED                 = 0x7d
+	CMD_RELOAD_REGDB                    = 0x7e
+	CMD_EXTERNAL_AUTH                   = 0x7f
+	CMD_STA_OPMODE_CHANGED              = 0x80
+	CMD_CONTROL_PORT_FRAME              = 0x81
+	__CMD_AFTER_LAST                    = 0x82
+	CMD_MAX                             = 0x83
```

Comment:
CMD_MAX should be CMD_AFTER_LAST-1

Reply:
You are right. My bad. It's been changed.